opengb/seed-core
# Shrink GRID logo

#### Any background context you want to provide?
The GRID logo in our customized version of SEED is too large.

#### What's this PR do?
This PR replaces the GRID logo with a version that is 20% of the old logo's size.

#### How should this be manually tested?
1. Start up SEED locally.
2. Request a password reset or create a new user with your email address.
3. Look for the email message in your email inbox.
4. The GRID logo at the end of the email message should be smaller than in previous versions.

#### What are the relevant tickets?
[Slack thread](https://opengb.slack.com/archives/CE007SA20/p1655153324846789)

#### Screenshots (if appropriate)
